AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -1981,15 +1981,16 @@ aternos.org#@##adUnit
 ||flashtalking.com/lgc|$empty,important,domain=aternos.org
 !#endif
 !#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android)
+@@||hbopenbid.pubmatic.com/translator$domain=aternos.org
+@@||prg.smartadserver.com/prebid/v1$domain=aternos.org
 @@||c.amazon-adsystem.com/bao-csm/aps-comm/aps_csm.js$domain=aternos.org
 @@||snigel-d.openx.net/*/arj|$domain=aternos.org
-@@||tlx.3lift.com/header/auction|$domain=aternos.org
+@@||tlx.3lift.com/header/auction$domain=aternos.org
 @@||as-sec.casalemedia.com/cygnus|$domain=aternos.org
-@@||fastlane.rubiconproject.com/a/api/fastlane.json|$domain=aternos.org
+@@||fastlane.rubiconproject.com/a/api/fastlane.json$domain=aternos.org
 @@||flashtalking.com/lgc|$domain=aternos.org
 @@||ib.adnxs.com/ut/v3/prebid|$domain=aternos.org
 @@||snigel-d.openx.net/w/1.0/arj?cc=1$domain=aternos.org
```

Comment:
On my end it's also nesessary to change this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/2fc76bae9c3d1949b9b279386d927022515afd03/AnnoyancesFilter/sections/antiadblock.txt#L1991
for example to something like this:
`@@||snigel-d.openx.net/w/1.0/arj?$domain=aternos.org`

By the way, changing this rule:
`aternos.org#%#AG_defineProperty('XMLHttpRequest.setUpdateListener', {value: function() { return; } });`
to something like:
```
aternos.org#%#AG_defineProperty('XMLHttpRequest.setUpdateListener', {value: function() { var whatever; } });
aternos.org#@%#AG_defineProperty('XMLHttpRequest.setUpdateListener', {value: function() { return; } });
```
also works, but I guess it's not needed.

Reply:
> On my end it's also nesessary to change this rule:

Now I see it too, and the new scriptlet works, thank you

Reply:
This `#%#` rule will not work in Firefox/Opera until new AdGuard Extension version will be released, so could you please also change this rule - `@@||snigel-d.openx.net/w/1.0/arj?cc=1$domain=aternos.org`?